src/rules/convert_require/rojo_sourcemap.rs

```diff
@@ -128,8 +164,25 @@ impl RojoSourcemap {
         let from_file = from_file.as_ref();
         let target_file = target_file.as_ref();
 
+        let binding = from_file.to_path_buf().join(target_file);
```

Comment:
This can be simplified to:
```suggestion
        let binding = from_file.join(target_file);
```
It avoids creating a temporary PathBuf that'll get discarded